.github/actions/base-ingest-cache/action.yml

```diff
@@ -18,7 +18,7 @@ runs:
         path: |
           .venv
           nltk_data
-        key: unstructured-ingest-${{ runner.os }}-${{ inputs.python-version }}-${{ hashFiles('requirements/ingest/*.txt') }}
+        key: unstructured-ingest-${{ runner.os }}-${{ inputs.python-version }}-${{ hashFiles('requirements/ingest/base.txt') }}
```

Comment:
isn't it safer to keep the glob, `*.txt`, here?

i would assume test.txt and base.txt should not change that much independently of each other anyway, but test.txt changing could impact fixture output in the worst case?